Traceback (most recent call last):
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/site-packages/nbclient/client.py", line 1314, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/asyncio/base_events.py", line 685, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/anaconda3/envs/pymc_env/lib/python3.12/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
stan_file <- "../code/rasch_model.stan"
mod <- cmdstan_model(stan_file)
mod$print()
------------------

----- stderr -----
fatal error: file '/opt/anaconda3/envs/stan_env/lib/clang/16/include/arm_fp16.h' has been modified since the precompiled header 'stan/src/stan/model/model_header_16_0.hpp.gch' was built: mtime changed (was 1706884841, now 1711057963)
note: please rebuild precompiled header 'stan/src/stan/model/model_header_16_0.hpp.gch'
----- stderr -----
Warning message:
“CmdStan's precompiled header (PCH) files may need to be rebuilt.
If your model failed to compile please run rebuild_cmdstan().
If the issue persists please open a bug report.”
----- stderr -----
1 error generated.
----- stderr -----
make: *** [make/program:61: /var/folders/s7/z86r4t9j6yx376cm120nln6w0000gn/T/RtmpDpA5WP/model-72122ecc4bf.o] Error 1
------------------

Error: An error occured during compilation! See the message above for more information.
Traceback:

1. cmdstan_model(stan_file)
2. CmdStanModel$new(stan_file = stan_file, exe_file = exe_file, 
 .     compile = compile, ...)
3. initialize(...)
4. self$compile(...)
5. stop(err_msg, call. = FALSE)

